src/types/constrainedEditor.d.ts

```diff
@@ -0,0 +1,59 @@
+declare module 'constrained-editor-plugin' {
```

Comment:
😮 Are they accepting contributions?

Reply:
The repository doesn't look active lately 😞 But we have already been using this in GCk6 and had types defined there